packages/lisk-cryptography/src/nacl/fast.ts

```diff
@@ -12,10 +12,11 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-// eslint-disable-next-line import/no-extraneous-dependencies
+// tslint:disable-next-line no-implicit-dependencies
 import sodium from 'sodium-native';
+import { NaclInterface } from './index';
```

Comment:
wouldn't this be circular dependency?

Reply:
As per my understanding, Typescript removes all import statements which only involve importing of interfaces and types. So, the transpiled code will not include this line. I can declare this interface in a separate file under types folder if you think that'd be better.

Reply:
yea it doesn't cause circular dependency on the built file, but i think it's good to avoid it anyway